modules/master-node-sizing.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id="master-node-sizing_{context}"]
+=  Master node sizing
+
+Control plane density focused tests were run at different node scale to determine the master node resource requirements depending on the number of nodes in the cluster. The recommended sizes are
```

Comment:
I'd replace this line with the following text:

>  The master node resource requirements depend on the number of nodes in the cluster. The following master node size recommendations are based on the results of control plane density focused testing.

Is this the number of master nodes or the number of total nodes?  If so, I'd clarify that in the table.